activerecord/lib/active_record/attribute_methods/time_zone_conversion.rb

```diff
@@ -98,7 +98,7 @@ def create_time_zone_conversion_attribute?(name, cast_type)
               still causes `String`s to be parsed as if they were in `Time.zone`,
               and `Time`s to be converted to `Time.zone`.
 
-              To keep the old behavior, you must add the following to your initializer:
+              To keep the old behavior, you must add the following to `config/application.rb`:
```

Comment:
Hey @flanger001 sorry for the delay. An initializer should work, and as Matthew said it could be because something in your app is loading AR early. Can you update this to say "in your initializer or in `config/application.rb`"?